pkg/util/util.go

```diff
@@ -63,3 +66,12 @@ func ParseEnv(p string, fs *flag.FlagSet) error {
 
 	return err
 }
+
+func IsTimescaleDBInstalled(conn pgxconn.PgxConn) bool {
+	var installed bool
+	err := conn.QueryRow(context.Background(), `SELECT count(*) > 0 FROM pg_extension where extname = 'timescaledb'`).Scan(&installed)
+	if err != nil {
+		return false
```

Comment:
I'd recommend at least logging this error

Reply:
I think it's better to keep without any log, as Postgres users will not be happy seeing any error log when they are using Plain Postgres purposefully.